usage/launcher/src/main/java/brooklyn/launcher/BrooklynLauncher.java

```diff
@@ -772,6 +782,7 @@ protected void startWebApps() {
             webServer.setPublicAddress(publicAddress);
             if (port!=null) webServer.setPort(port);
             if (useHttps!=null) webServer.setHttpsEnabled(useHttps);
+            webServer.setShutdownRequestListener(shutdownRequestListener);
```

Comment:
It's a simple setter which for me assumes `@Nullable`. I've already marked the provider constructor. Added the annotation here as well though, can't hurt.